project/BuildSettings.scala

```diff
@@ -109,7 +109,13 @@ object Doc extends AutoPlugin {
 
   lazy val scalaDocSettings = Seq(
     autoAPIMappings := true,
-    apiURL := Some(url(s"http://api.silhouette.rocks/${version.value}/"))
+    apiURL := Some(url(s"http://api.silhouette.rocks/${version.value}/")),
+    apiMappings ++= {
+      implicit val cp = (fullClasspath in Compile).value
+      Map(
+        scalaInstance.value.libraryJar -> url(s"http://www.scala-lang.org/api/${scalaVersion.value}/")
```

Comment:
@akkie can you help me? I've added scala api, but I always get this error
```
[error] /home/travis/build/minutemen/silhouette/silhouette/src/main/scala/silhouette/http/decoder/package.scala:33: Could not find any member to link for "scala.util.Failure".
```
[see detail](https://travis-ci.org/minutemen/silhouette/jobs/192813465#L1033)

I've copied it from [silhouette-akka-http](https://github.com/minutemen/silhouette-akka-http/blob/master/project/BuildSettings.scala#L110)